### ISSUE TITLE
OSError: libcublas.so.10.0: cannot open shared object file: No such file or directory

### ISSUE THREAD

**Issue by leimao** [microsoft/onnxruntime#2367]
**Describe the bug**

ONNXRuntime-GPU 1.0.0 is not compatible with CUDA 10.1 but ONNXRuntime-GPU 0.5.0 is.

**Urgency**
If there are particular important use cases blocked by this or strict project-related timelines, please share more information and dates. If there are no hard deadlines, please specify none.

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): 18.04.3 LTS
- ONNX Runtime installed from (source or binary): binary
- ONNX Runtime version: 1.0.0
- Python version: 3.6.
- Visual Studio version (if applicable):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version: 10.1
- GPU model and memory: V100

**To Reproduce**
Describe steps/code to reproduce the behavior:

```
root@6d65123700ea:/mnt# pip install onnxruntime-gpu==1.0.0
Collecting onnxruntime-gpu==1.0.0
  Using cached https://files.pythonhosted.org/packages/c0/47/2406d03b607bc2f1852ca885cf8bee4a5b3b6eaa2e83f364085141f5bafd/onnxruntime_gpu-1.0.0-cp36-cp36m-manylinux2010_x86_64.whl
Installing collected packages: onnxruntime-gpu
Successfully installed onnxruntime-gpu-1.0.0
WARNING: You are using pip version 19.2.3, however version 19.3.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
root@6d65123700ea:/mnt# python
Python 3.6.8 (default, Aug 20 2019, 17:12:48) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import onnxruntime
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.6/dist-packages/onnxruntime/__init__.py", line 12, in <module>
    from onnxruntime.capi._pybind_state import get_all_providers, get_available_providers, get_device, RunOptions, SessionOptions, set_default_logger_severity, NodeArg, ModelMetadata, GraphOptimizationLevel, ExecutionMode
  File "/usr/local/lib/python3.6/dist-packages/onnxruntime/capi/_pybind_state.py", line 8, in <module>
    import onnxruntime.capi._ld_preload
  File "/usr/local/lib/python3.6/dist-packages/onnxruntime/capi/_ld_preload.py", line 12, in <module>
    _libcublas = CDLL("libcublas.so.10.0", mode=RTLD_GLOBAL)
  File "/usr/lib/python3.6/ctypes/__init__.py", line 348, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: libcublas.so.10.0: cannot open shared object file: No such file or directory
>>> 
```
```
root@6d65123700ea:/mnt# pip uninstall onnxruntime-gpu==1.0.0
Uninstalling onnxruntime-gpu-1.0.0:
  Would remove:
    /usr/local/bin/onnxruntime_test
    /usr/local/lib/python3.6/dist-packages/onnxruntime/*
    /usr/local/lib/python3.6/dist-packages/onnxruntime_gpu-1.0.0.dist-info/*
Proceed (y/n)? y
  Successfully uninstalled onnxruntime-gpu-1.0.0
root@6d65123700ea:/mnt# pip install onnxruntime-gpu==0.5.0
Collecting onnxruntime-gpu==0.5.0
  Downloading https://files.pythonhosted.org/packages/c2/6d/617eaa9617284ad34165ebab5b5996b1ed38afaa2f5fbba56d05a7d97ac3/onnxruntime_gpu-0.5.0-cp36-cp36m-manylinux2010_x86_64.whl (5.6MB)
     |████████████████████████████████| 5.6MB 5.6MB/s 
Installing collected packages: onnxruntime-gpu
Successfully installed onnxruntime-gpu-0.5.0
WARNING: You are using pip version 19.2.3, however version 19.3.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
root@6d65123700ea:/mnt# python
Python 3.6.8 (default, Aug 20 2019, 17:12:48) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import onnxruntime
>>> 
```

**Expected behavior**

ONNXRuntime 0.5.0 has no problem for CUDA 10.1. But ONNXRuntime 1.0.0 does not seem to work with CUDA 10.1.

**Screenshots**

**Additional context**
Add any other context about the problem here. If the issue is about a particular model, please share the model details as well to facilitate debugging.

**Comment by pranavsharma** [comment]
1.0 doesn't support cuda 10.1. This was a deliberate decision to keep things consistent. Support for a 10.1 in python Linux 0.5 was an oversight that we corrected in 1.0 release (all other 0.5 packages supported 10.0 only).

**Comment by leimao** [comment]
> 1.0 doesn't support cuda 10.1. This was a deliberate decision to keep things consistent. Support for a 10.1 in python Linux 0.5 was an oversight that we corrected in 1.0 release (all other 0.5 packages supported 10.0 only).

Thanks. But if I compile ONNXRuntime-GPU using CUDA 10.1, it should work well right?

**Comment by pranavsharma** [comment]
> > 1.0 doesn't support cuda 10.1. This was a deliberate decision to keep things consistent. Support for a 10.1 in python Linux 0.5 was an oversight that we corrected in 1.0 release (all other 0.5 packages supported 10.0 only).
> 
> Thanks. But if I compile ONNXRuntime-GPU using CUDA 10.1, it should work well right?

Yes.

**Comment by leimao** [comment]
> > > 1.0 doesn't support cuda 10.1. This was a deliberate decision to keep things consistent. Support for a 10.1 in python Linux 0.5 was an oversight that we corrected in 1.0 release (all other 0.5 packages supported 10.0 only).
> > 
> > 
> > Thanks. But if I compile ONNXRuntime-GPU using CUDA 10.1, it should work well right?
> 
> Yes.

Thank you.
